src/packages/core/workspace/components/variant-selector/variant-selector.element.ts

```diff
@@ -232,7 +243,9 @@ export class UmbVariantSelectorElement extends UmbLitElement {
 																? html`<uui-icon class="add-icon" name="icon-add"></uui-icon>`
 																: nothing}
 															<div>
-																${variant.name} <i>(${variant.culture})</i> ${variant.segment}
+																${variant.name ||
+																this._languages.getValue().find((x) => x.unique === variant.culture)?.name}
+																<i>(${variant.culture})</i> ${variant.segment}
 																<div class="variant-selector-state">${variant.state}</div>
 															</div>
 														</button>
```

Comment:
I would have wanted to put this comment on the static styles prop, but couldnt find out how.

I know this is not part of this PR. but I noticed it, so I'm going to mention it.

The styling of this component sets the font-family it self. It should not do that instead it should import and use the UmbTextStyle css :-)